.changeset/strange-pugs-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/commercial': minor
+---
+
+Remove google analytics from Web and AMP on the Guardian's main domain
```

Comment:
How about
```suggestion
Stop sending commercial-related events to Google Analytics
```

Reply:
V.minor but it might be a little more accurate to only refer to commercial for this PR. Something like:

"Prepare code to remove GA from Commercial"

Reply:
Ok Ravi I've changed that in the editor in here.